desktop-src/Sync/slim-reader-writer--srw--locks.md

```diff
@@ -16,6 +16,8 @@ SRW locks provide two modes in which threads can access a shared resource:
 
 -   **Shared mode**, which grants shared read-only access to multiple reader threads, which enables them to read data from the shared resource concurrently. If read operations exceed write operations, this concurrency increases performance and throughput compared to critical sections.
 -   **Exclusive mode**, which grants read/write access to one writer thread at a time. When the lock has been acquired in exclusive mode, no other thread can access the shared resource until the writer releases the lock.
+    > [!NOTE]
+    > Exclusive mode SRW locks are not re-entrant. If a thread tries to acquire a lock that it already holds, that attempt will fail (for [**TryAcquireSRWLockExclusive**](https://msdn.microsoft.com/library/Dd405523(v=VS.85).aspx)) or deadlock (for [**AcquireSRWLockExclusive**](https://msdn.microsoft.com/library/ms681930(v=VS.85).aspx))
```

Comment:
I'm not sure reentrant is the right term here.
People like to say "recursive", but I'm not sure that is right either.

Reply:
Thanks for your feedback. I revised the text from "...are not re-entrant" to "...cannot be acquired recursively". This change should be published to the live docs later today.